nototools/generate_website_2_data.py

```diff
@@ -678,13 +725,16 @@ def build_zips(self, families):
   def build_universal_zips(self, families):
     hinted_fonts = []
     unhinted_fonts = []
+    readme_path = self.get_readme_path('all')
     for family_data in families.values():
       hinted_fonts.extend(
           family_data.hinted_members or family_data.unhinted_members)
```

Comment:
Hmmm.  It has for a few years now, I think.  It's not so much "these are the hinted fonts" as it is "these are fonts we suggest if your platform does better with hinted fonts".  So it provides hinted fonts where we have them, and unhinted if not, so that it provides the same coverage.  This does mean the file name is misleading.

I think originally we had three sets of fonts and one was named for windows, which is arguably cleaner naming.